.github/workflows/manual-cherrypick.yml

```diff
@@ -0,0 +1,82 @@
+on: 
```

Comment:
Please move this to one of the repositories involved in the CP process.

Reply:
Sure, closed pull request.

umm yeah @CaseyHillers suggested the cherry pick workflow to live on cocoon for other repos to consume. Personally I am good with the manual workflows (workflow that takes inputs) to live on any repo. 
umm yeah might be a good idea to quickly sync on this to see how to proceed